docker/test/integration/minifi/__init__.py

```diff
@@ -294,7 +314,7 @@ def __exit__(self, exc_type, exc_val, exc_tb):
             container.remove(v=True, force=True)
 
         # Clean up images
-        for image in self.images:
+        for image in reversed(self.images):
```

Comment:
What's the goal of this change?

Reply:
Since kafka-consumer images are based on kafka-broker image, we need to rm the consumer images before removing the broker one. Thus, the last image should be removed first, in general.